### ISSUE TITLE
Training loss always reported as 0 for SCNet

### ISSUE THREAD

**Issue by 7eraflops** [ZFTurbo/Music-Source-Separation-Training#203]
I recently tried training and ```scnet``` model using this repository (specifically the SCNet XL model provided in the pre-trained models section.) I used the provided checkpoint and config, and only adjusted the batch size and gradient accumulation steps to fit everything in memory. The training works correctly, but always reports loss and avg_loss as 0. This isn't the case for other models I tried training like ```bs_roformer``` or ```htdemucs```

Here's the exact command I used:
```
python train.py \
          --model_type scnet \
          --config_path /home/teraflops/Documents/code/music_source_separation/configs/scnet_xl/config_musdb18_scnet_xl.yaml \
          --start_check_point /home/teraflops/Documents/code/music_source_separation/checkpoints/scnet_xl/model_scnet_ep_54_sdr_9.8051.ckpt \
          --results_path /home/teraflops/Documents/code/music_source_separation/results/scnet_xl/ \
          --data_path /home/teraflops/Documents/code/music_source_separation/datasets/moisesdb_musdb18_style_split/train/ \
          --valid_path /home/teraflops/Documents/code/music_source_separation/datasets/moisesdb_musdb18_style_split/valid/ \
          --num_workers 4 \
          --device_ids 0 \
          --wandb_key <my_wandb_key>
```

Which gave these results in wandb:

<img width="2925" height="997" alt="Image" src="https://github.com/user-attachments/assets/488c94e7-d58a-44a0-819f-b96d1c94c6a2" />

SDR is calculated correctly for validation, but loss is always 0. Am I doing something wrong?

**Comment by jarredou** [comment]
The default loss can return 0 when batch_size is small, try with l1_loss (or spec_rmse_loss, which is the one used by original scnet implementation)

**Comment by ZFTurbo** [comment]
Standard loss requires batch size larger than 1. It should work ok with 2.
